manipulation/src/functTest/resources/mongo-java-driver/gradle/publish.gradle

```diff
@@ -102,7 +102,7 @@ configure(javaProjects) { project ->
 
     afterEvaluate {
         jar configureJarManifestAttributes(project)
-        publishing.publications.mavenJava.artifactId = project.archivesBaseName
+//        publishing.publications.mavenJava.artifactId = project.archivesBaseName
```

Comment:
What was this originally doing?

Reply:
This was for that project setting the artifactId to correspond to instructions in https://docs.gradle.org/current/userguide/maven_plugin.html#uploading_to_maven_repositories However, I wanted to verify that the changes I made could handle the situation where this _wasn't_ done hence commenting it out.